specs/language/conversions.tex

```diff
@@ -118,6 +118,19 @@
 of boolean type. A zero value is converted to \texttt{false}; all other values
 are converted to \texttt{true}.
 
+\Sec{Flat conversion}{Conv.flat}
+
+\p There is a flattened order of subobjects for an aggregate, \texttt{A},
```

Comment:
I think this language is fine for now, but we should probably have some common definitions and unify this with the language in [*Decl.Init.Agg*](https://microsoft.github.io/hlsl-specs/specs/hlsl.html#Decl.Init.Agg).

I filed #372 to track that separately.

Reply:
Should this be changed to 'Elementwise conversion'?